tests/src/readytorun/main.cs

```diff
@@ -310,6 +318,11 @@ static void RunAllTests()
 
     static int Main()
     {
+        LLILCJitEnabled =
```

Comment:
Maybe nice to add a comment here - that this should be removed once exceptions work in LLILC